fastly/resource_fastly_service_v1_test.go

```diff
@@ -551,6 +551,42 @@ func TestAccFastlyServiceV1_createDefaultTTL(t *testing.T) {
 	})
 }
 
+func TestAccFastlyServiceV1_brokenSnippet(t *testing.T) {
```

Comment:
@bengesoff Could add a doc string that just gives a summary of what the test is validating for future travellers.

Tests are written at a time when the full context of what is going on is still fresh, but if I was to come back to this test it would be hard for me to know what it was trying to validate and why.

So something like…

```go
// TestAccFastlyServiceV1_brokenSnippet validates X and it does this by doing Y first, then Z and ensuring ABC happens.
```